snapcraft.yaml

```diff
@@ -1382,9 +1382,105 @@ parts:
         sed -i 's#libdir=/usr#libdir=${prefix}#' $PC
         sed -i 's#includedir=/usr#includedir=${prefix}#' $PC
       done
-
+  libicuc:
+    source: https://github.com/unicode-org/icu.git
+    source-tag: release-73-1
+#   ext:updatesnap
+#   version-format:
+#     format: 'release-%M-%m'
+    source-subdir: icu4c/source
+    plugin: autotools
+    autotools-configure-parameters:
+      # WORKAROUND: Install to /usr instead of /usr/local because it's not in search paths
+      - --prefix=/usr
+      - --disable-renaming
+      - --libdir=\${exec_prefix}/lib/$CRAFT_ARCH_TRIPLET
+    build-packages:
+      - doxygen
+  gi-docgen:
+    source: https://gitlab.gnome.org/GNOME/gi-docgen.git
+    source-tag: '2023.1'
+    plugin: python
+    build-environment:
+      # WORKAROUND: The python plugin is broken with gnome extension
+      - PATH: ${CRAFT_PART_INSTALL}/bin:${PATH}
+      - PYTHONPATH: ""
+    stage:
+      # WORKAROUND: Skip venv from python plugin
+      - -bin/activate
+      - -bin/activate.csh
+      - -bin/activate.fish
+      - -bin/Activate.ps1
+      - -bin/python
+      - -bin/python3
+      - -bin/python3.10
+      - -bin/pip
+      - -bin/pip3
+      - -bin/pip3.10
+      - -pyvenv.cfg  
+  webkitgtk:
+    after: [gi-docgen, libicuc]
+    source: https://salsa.debian.org/webkit-team/webkit.git
+    source-tag: upstream/2.40.1
```

Comment:
Mmm... but webkitgtk 2.40 is already being included...

![image](https://github.com/ubuntu/gnome-sdk/assets/105285003/0fdc6e80-0ecc-4063-bd42-c06bddaa77e9)

Reply:
Or are you just adding this to "jump" to version 2.50 as soon as it goes out?